src/unzipObjWith.js

```diff
@@ -0,0 +1,45 @@
+import {
+  curryN,
+  compose,
+  pipe,
+  apply,
+  map,
+  toPairs,
+  flip,
+  transpose,
+} from 'ramda';
+
+/**
+ * Unzips an object into an two arrays of transformed keys and transformed values.
+ *
+ * When executed it converts the object into value/key pairs which are passed to the transformation
+ * functor. The returned, transformed, `[key, value]` pairs are then grouped into two arrays of
```

Comment:
The description is way to complicated even for me to understand (and I know what the implementation should do). What about:
```
Creates a new list out of the supplied object by applying the function to each key/value pairing. 
```

Reply:
Thanks- I really struggled to think of a good way of summing it up!